h/templates/activity/search.html.jinja2

```diff
@@ -105,7 +136,7 @@
     </ol>
 
     {% if group %}
-    <aside class="search-result-sidebar">
+    <aside class="search-result-sidebar {%- if not more_info %} search-result-hide-on-small-screens{% endif %}">
```

Comment:
I don't know yet whether we're gonna want to change how the list of members is computed. The sidebar could certainly become a simple macro already though. Although yes - follow up PR. We're accruing several refactorings that I want to do in follow-up PRs, once the rush to get this feature complete and into people's hands for testing is over